packages/netlify-cms-backend-bitbucket/src/implementation.ts

```diff
@@ -255,27 +254,67 @@ export default class BitbucketBackend implements Implementation {
     const listFiles = () =>
       this.api!.listFiles(folder, depth).then(({ entries, cursor: c }) => {
         cursor = c.mergeMeta({ extension });
-        return filterByPropExtension(extension, 'path')(entries);
+        return entries.filter(e => filterByExtension(e, extension));
```

Comment:
`filterByPropExtension` was a generic method to filter arrays using any prop by extension. As it was only used to filter by file paths, I replaced is with a simple filter function.